src/nativestore/NodeManager.cpp

```diff
@@ -344,7 +343,7 @@ void NodeManager::persistNodeIndex() {
             node_manager_logger.error("Node label/ID is longer ( " +
                                       std::to_string(this->nodeIndex.begin()->first.length()) +
                                       " ) than the index key size " + std::to_string(NodeManager::INDEX_KEY_SIZE));
-            throw "Node label/ID is longer than the index key size!";
+            throw std::logic_error("Node label/ID is longer than the index key size!");
```

Comment:
Lets write this to a logger instead of throwing the error

Reply:
Replaced with logger

Reply:
Ok